settings.gradle.kts

```diff
@@ -3,6 +3,7 @@ rootProject.name = "tractusx-edc"
 include(":edc-extensions:business-partner-validation")
 include(":edc-extensions:control-plane-adapter")
 include(":edc-extensions:cx-oauth2")
+include(":edc-extensions:vault-memory")
```

Comment:
Yes, I would highly agree to move it into the `edc-runtime-memory` module. If we keep it as an extension we would also publish this as a jar which mean people will maybe us it, which again means we need to maintain/support/update in a good shape. Of course we also need to maintain/support/update the `edc-runtime-memory` but I think it's less work for us

Reply:
yep, already done